core/services/pipeline/runner.go

```diff
@@ -8,49 +8,53 @@ import (
 	"sync"
 	"time"
 
-	"gopkg.in/guregu/null.v4"
-
 	"github.com/jpillora/backoff"
-	"github.com/smartcontractkit/chainlink/core/service"
-	"github.com/smartcontractkit/chainlink/core/store/models"
-
 	"github.com/pkg/errors"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promauto"
 	"github.com/smartcontractkit/chainlink/core/logger"
 	"github.com/smartcontractkit/chainlink/core/utils"
+	"gopkg.in/guregu/null.v4"
 	"gorm.io/gorm"
+
+	"github.com/smartcontractkit/chainlink/core/service"
+	"github.com/smartcontractkit/chainlink/core/services/eth"
+	"github.com/smartcontractkit/chainlink/core/store/models"
 )
 
 //go:generate mockery --name Runner --output ./mocks/ --case=underscore
 
-type Runner interface {
-	service.Service
+type (
```

Comment:
nit: didn't we get to the bottom of this earlier in that doing the big type def is not idiomatic go?